src/App.js

```diff
@@ -9,7 +12,13 @@ const article = {
 };
 
 const App = () => (
-  <h1>React Article</h1>
+  <div className="App">
+    <Article
+      title={article.title}
+      text={article.text}
+      date={article.date}
+    />
```

Comment:
Good, but you can simplify it
```suggestion
    <Article
      {...article}
    />
```